app/products/calls/components/current_call/current_call.tsx

```diff
@@ -129,23 +145,16 @@ const CurrentCall = (props: Props) => {
         <View style={style.wrapper}>
             <View style={style.container}>
                 <CallAvatar
-                    userId={props.speaker?.id}
-                    volume={props.speaker?.isTalking ? 0.5 : 0}
+                    userId={speaker ? speaker.id : ''}
+                    volume={speaker && speaker.id ? 0.5 : 0}
                 />
                 <View style={style.userInfo}>
                     <Text style={style.speakingUser}>
-                        <FormattedText
-                            id='current_call.user-is-speaking'
-                            defaultMessage='{username} is speaking'
-                            values={{username: displayUsername(props.speakerUser, props.teammateNameDisplay)}}
-                        />
+                        {speaker && `${displayUsername(speaker, props.teammateNameDisplay)} is talking`}
+                        {!speaker && 'No one is talking'}
                     </Text>
                     <Text style={style.currentChannel}>
-                        <FormattedText
-                            id='current_call.channel-name'
-                            defaultMessage='~{channelName}'
-                            values={{channelName: props.channel.display_name}}
-                        />
+                        {`~${props.channel.display_name}`}
```

Comment:
is display name set for GM and DM channels ?

Reply:
We haven't yet decided how to handle this from a UI perspective. The web plugin has this issue as well.

Reply:
I guess this will be handled on a later PR once decided then